src/main.cpp

```diff
@@ -263,151 +268,166 @@ int main(const int argc, const char *argv[])
 
     time_tracker.start("Integration Inset " + inset_pos);
 
+    // Print initial values, and initial progress (0)
+    progress_tracker.print_progress_mid_integration(inset_state);
+
     // Start map integration
     while (inset_state.n_finished_integrations() < max_integrations &&
-           (inset_state.max_area_error().value > max_permitted_area_error ||
-            std::abs(inset_state.area_drift() - 1.0) > 0.01)) {
-
+            (inset_state.max_area_error(false).value > max_permitted_area_error ||
+            std::abs(inset_state.area_expansion_factor() - 1.0) > max_permitted_area_expansion)) {
```

Comment:
I feel if this logic is moved into a method inside `inset_state`, the code could become cleaner. In that way, in the future, we can also do more complicated conditions for the termination of the integration.

Reply:
I think you are right. I'll incorporate this change, along with not printing the area drift unnecessarily.
`max_area_error` also probably should not print.